backend/drone.py

```diff
@@ -43,11 +45,10 @@ def disarm_vehicle(self):
         """
 
         self.vehicle.armed = False
-        while self.vehicle.armed==True:
+        while self.vehicle.armed:
             print('Waiting for the drone to disarm.')
             time.sleep(1)
-        if not self.vehicle.armed:
-            return True
+        return True
```

Comment:
The modifications made are not appropriate. Please try to implement new functions instead of making unnecessary edits to already tested functions.